integration_tests/integration/uploaded-documents-missing.spec.js

```diff
@@ -103,37 +102,6 @@ context('Missing uploaded documents', () => {
     })
   })
 
```

Comment:
do we need a test to verify that the entry is now empty upon previous mdr?

Reply:
Added an integration test